cms/cms.go

```diff
@@ -149,31 +138,41 @@ func DownloadContentZip(target string, downloader Downloader) error {
 		return nil
 	}
 
-	downloadTargetFile, err := os.Create(target)
+	log.Info("downloading %s from S3 bucket", zipName)
+
+	var numBytes int64
+
+	err := checkLsExists()
 	if err != nil {
-		errors.Errorf("Unable to open file %q, %v", target, err)
+		return err
 	}
 
-	defer downloadTargetFile.Close()
+	path := "s3://" + bucketName + "/" + zipName
```

Comment:
Why do we use a command line instead of the downloader? (I think using the Downloader would be more robust, as we wouldn't depend on running external commands)

Reply:
The S3 downloader library in the code is deprecated and fixing it would be way beyond requirements.
Did the code work for you ?
Also, we don’t have the account id’s and access codes in our environments any more for the new environments … so, it’s not clear to me how to overcome this problem. (edited) 
The code also checks to see if the aws command is available.